File: hw1/driver.py
import linkedlist

def main():
	ll = linkedlist.LinkedList()

	for i in xrange(50):
		ll.append("test")
	ll.append("hai")
	ll.append("test")

	ll.populate("db.txt", "\n")

	print ll


if __name__ == "__main__":
	main()

File: hw12/driver.js
var LinkedList = require('./linkedlist').LinkedList;

function main() {
    var list = new LinkedList();

    list.populate('db.txt', '\n').then(function printIt() {
        console.log(list.toString());
    }).then(function findSomething() {
        console.log(list.find('Weaver of Lies'));
        console.log(list.search('Anowon, the Ruin Sage'));
    });
}

main();